### ISSUE TITLE
Application does not remember window size on Windows

### ISSUE THREAD

**Issue by Loilo** [keeweb/keeweb#586]
I recently started using KeeWeb on Windows (where I used the original KeePass until then). It works pretty fine, I'm just annoyed by one thing:

KeeWeb does not seem to remember window size and position over the course of multiple sessions (i.e. close - open). This does not apply to recovering the window from tray.

**Comment by antelle** [comment]
I just tried it, and it seems to be ok on Windows 10 and latest keeweb.

**Comment by Loilo** [comment]
Okay, I investigated on this a little, and I was completely wrong about the cause. The issue actually has nothing to do with remembering size across sessions. It's about Auto Type.

**If the KeeWeb window was maximized, it un-maximizes by triggering the global Auto Type shortcut.**

Then, after using Auto Type when revisiting KeeWeb, it won't go back to its original, maximized state. Instead it will stay unmaximized in the top left corner of the screen—a seemingly fixed size and place where it went when it un-maximized.

Sorry for the confusion. :)

**Comment by antelle** [comment]
You're right, we don't handle maximized states in all places, but it's a bit another issue.

**Comment by Loilo** [comment]
Totally. My bad for not observing thoroughly enough.

**Comment by antelle** [comment]
Thank you for spending time to investigate it!

**Comment by Miltora** [comment]
I have a similiar issue in this topic. I created a shortcut and in the properties I defined run minimised but it won't do it. I tested it with other programs and it works just fine. Is it just on my computer?

**Comment by evlo** [comment]
It does not remember the size even when it was never maximized. It just opens once in last size and next time it is wider then the screen so not even close window buttons are visible etc.
